doc/build-unix.md

```diff
@@ -204,7 +204,7 @@ If you need to build Boost yourself:
 
 	sudo su
 	./bootstrap.sh
-	./bjam install
+	./b2 install
```

Comment:
We should probably remove this all together.  Boost install instructions are available elsewhere.  We shouldn't be maintaining them.

Reply:
Agreed